spec/services/coronavirus/pages/content_builder_spec.rb

```diff
@@ -87,12 +87,6 @@
       expect(subject.sub_sections_data).to eq []
     end
 
-    it "raises an error if the sub-section has invalid content" do
-      create(:coronavirus_sub_section, page: page, content: "I am invalid")
-
-      expect { subject.sub_sections_data }.to raise_error(Coronavirus::Pages::ContentBuilder::InvalidContentError)
```

Comment:
The exception definition for `InvalidContentError` needs to be removed as well: https://github.com/alphagov/collections-publisher/blob/23e7800a50ff4848c71b5e7baa78653cd50bac11/app/services/coronavirus/pages/content_builder.rb#L5

As does the rescuing and re-throwing of the exception: https://github.com/alphagov/collections-publisher/blob/23e7800a50ff4848c71b5e7baa78653cd50bac11/app/services/coronavirus/pages/content_builder.rb#L84-L85

Reply:
good spot. Thanks :)